workers/tasks/import/rules.ts

```diff
@@ -1,49 +1,50 @@
-import { buildRuleset, transformers, transformUtil } from '@pubpub/prosemirror-pandoc';
-import { defaultNodes, defaultMarks } from 'components/Editor/schemas';
-
 import * as katex from 'katex';
```

Comment:
This file is actually an important part of the changeset despite being hidden by default from Github.